igvm/settings.py

```diff
@@ -162,6 +171,108 @@
     {'hypervisor': HYPERVISOR_ATTRIBUTES},
 ]
 
+AWS_RETURN_CODES = {
+    'pending': 0,
+    'running': 16,
+    'shutting-down': 32,
+    'terminated': 48,
+    'stopping': 64,
+    'stopped': 80,
+}
+
+AWS_CONFIG = [
+    {
+        'puppet': {
+            'ca_addr': '212.53.146.132',
+            'master_addr': '212.53.146.131',
```

Comment:
Still not using dns names, why?

Reply:
Because this is just a temporarily solution using our deprecated puppet infrastructure. The plan is  Furthermore we have to use internal name with external address here. We will get rid of this soonish.

Reply:
Ah, really? I thought we already were using a public name with the old vm_externals. Okay then, but maybe add a FIXME here.

Reply:
I've added a TODO